docs/rules/v-slot-style.md

```diff
@@ -0,0 +1,112 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/v-slot-style
+description: enforce `v-slot` directive style
+---
+# vue/v-slot-style
+> enforce `v-slot` directive style
+
+- :wrench: The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) can automatically fix some of the problems reported by this rule.
+
+## :book: Rule Details
+
+This rule enforces `v-slot` directive style which you should use shorthand or long form.
+
+<eslint-code-block fix :rules="{'vue/v-slot-style': ['error']}">
+
+```vue
+<template>
+  <!-- ✓ GOOD -->
+  <my-component v-slot="data">
+    {{data}}
+  </my-component>
+  <my-component>
+    <template #default>content</template>
+    <template #one>content</template>
+    <template #two>content</template>
+  </my-component>
+
+  <!-- ✗ BAD -->
+  <my-component #default="data">
+    {{data}}
+  </my-component>
+  <my-component>
+    <template v-slot>content</template>
+    <template v-slot:one>content</template>
+    <template v-slot:two>content</template>
+  </my-component>
+</template>
+```
+
+</eslint-code-block>
+
+## :wrench: Options
+
+Default is set to `shorthand`.
```

Comment:
The default for `atComponent` is `v-slot`.

Reply:
Oh, good catch!